packages/gasket-plugin-log/lib/index.js

```diff
@@ -56,6 +56,29 @@ module.exports = {
           description: 'Setup Winston log transports',
           link: 'README.md#logTransports',
           parent: 'init'
+        }],
+        configurations: [{
+          name: 'log',
+          link: 'README.md#configuration',
+          description: 'Setup and customize log logger',
+          type: 'object',
+          default: '{}'
+        }, {
+          name: 'log.prefix',
+          link: 'README.md#configuration',
+          description: 'Used to set the prefix in the winston format.',
+          type: 'string'
+        }, {
+          name: 'winston',
+          link: 'README.md#configuration',
+          description: 'Setup and customize winston logger.',
+          type: 'object',
+          default: '{}'
+        }, {
+          name: 'winston.level',
+          link: 'README.md#configuration',
+          description: 'Type or level of logger.',
+          type: 'string'
```

Comment:
I don't think we need these details in the index as it's an incomplete picture of the winston options
```suggestion
```